js/bytex.js

```diff
@@ -797,11 +912,7 @@ module.exports = class bytex extends Exchange {
         //                     "exchange_id": 176
         //                 }
         //             }
```

Comment:
Can you move this response one tab space to the left? So that it aligns with other response comments?